### DISCUSSION TITLE
grafana-toolkit build needs more memory

### DISCUSSION THREAD

**Discussion by ptallettms** [grafana/grafana#47856] [Plugins]
I am using a large library in my plugins and the build fails as its out of memory or takes 2 hours to complete.

How do I give the build more memory to work with?

Cheers,
Paul

**Comment by ptallettms** [comment]
Or is there a workaround (eg using Webpack directly)?

**Comment by leeoniya** [comment]
maybe https://support.circleci.com/hc/en-us/articles/360009208393-How-Can-I-Increase-the-Max-Memory-for-Node- ?

**Reply by ptallettms** [reply]
Thanks. Yes, that's what we do in our non-Grafana build, but I don;t see how to pass this on the grafana-toolkit command (even reading through the source)

**Reply by ptallettms** [reply]
Oh, didn't see the ENV option. Trying that now

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
